cognee/tests/unit/run_tasks/conftest.py

```
@@ -0,0 +1,18 @@
+from cognee.modules.users.methods.get_default_user import get_default_user
+import cognee
+
+
+@pytest.fixture(autouse=True, scope="session")
+def set_get_mock_user_wrapper():
+
+    def get_mock_user_wrapper():
+        warnings.warn("\n\n\n---------------USING MOCK USER--------------------\n\n\n")
+        return get_mock_user(None, None)
+
+    get_default_user = cognee.modules.users.methods.get_default_user
```

Ruff

Redefinition of unused `get_default_user` from line 4

(F811)